index.html

```diff
@@ -4,6 +4,7 @@
     <title>welcome to XYZ</title>
   </head>
   <body>
+  <h1>Hey Customer! Welcome to our shop2</h1>
     
```

Comment:
This is good but you add a message, "we are ready to serve you"

Reply:
great from leader